src/sentry/static/sentry/app/components/activity/noteInput.jsx

```diff
@@ -182,10 +191,26 @@ const NoteInput = React.createClass({
     this.finish();
   },
 
+  onAdd(id, display) {
+    let mentions = this.state.mentions;
+    mentions.push([id, display]);
+    this.setState({mentions: mentions});
```

Comment:
In React, it is ideal to just re-create the `mentions` array here rather than append to it. I can talk about this in person.

Also, if the key and variable name are the same, you can use this shorthand:

```javascript
let mentions= this.state.mentions.concat([id, display]);
this.setState({mentions});
```